### PR TITLE
fix: make features package independent of auth

### PR DESCRIPTION
**Pull request by karolsojko**
as the only thing that `features` package is depending on in `auth` is the permissions - which is a descriptor of a feature, it makes more sense to move the permissions model to the features package to make it independent of auth and avoid potential future circular dependencies

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2491,6 +2491,11 @@
     "@typescript-eslint/eslint-plugin" "^4.14.0"
     "@typescript-eslint/parser" "^4.14.0"
 
+"@standardnotes/features@1.0.0":
```
Comment (gorjan5sk) [inline yarn.lock:2494]: I'm guessing we'll update the version when the dependents are updated?

Comment (karolsojko) [reply yarn.lock:2494]: this is for `@standardnotes/snjs` package ... no need to update the features dependcy version in there yet, but we can do it soon

**Comment by vardan-arm** [comment]
Shouldn't we also remove Permissions-related files from `packages/auth/src`?

**Comment by karolsojko** [comment]
@vardan-arm yes, coming in next PR :)

**Pull request merged**